docs/beyond_basics/101-182-catalog.rst

```diff
@@ -3,7 +3,573 @@
+``dataset_version`` (``-v/--dataset-version`` flag):
+
+.. runrecord:: _examples/DL-101-182-109
+   :language: console
+   :workdir: catalog
+   :cast: catalog_basics
+   :notes: Add a superdataset to the catalog
+
+   $ datalad catalog set-super --catalog-dir data-cat --dataset-id abcd --dataset-version 1234
+
+The catalog ``set-super(ok)`` result shows that the superdataset was successfully set
+for the catalog, and you will now also be able to see an additional ``super.json`` file in the
+catalog metadata directory. The content of this file is a simple JSON object specifying the
+superdataset's ``dataset_id`` and ``dataset_version``:
+
+.. runrecord:: _examples/DL-101-182-110
+   :language: console
+   :workdir: catalog
+   :cast: catalog_basics
+   :notes: Display the content of super.json
+
+   $ cat data-cat/metadata/super.json | jq .
+
+*Now*, when one navigates to the catalog's root address without a ``dataset_id`` and
+``dataset_version`` specified in the URL, the browser application will find that a
+default dataset is indeed specified for the catalog, and it will navigate to that specific
+dataset view!
+
+Catalog configuration
+"""""""""""""""""""""
+
+A useful feature of the catalog process is to be able to configure certain properties according
+to your preferences. This is done with help of a config file (in either ``JSON`` or ``YAML`` format)
+and the ``-y/--config-file`` flag during catalog generation. DataLad Catalog provides a default
+config file with the following content:
+
+.. runrecord:: _examples/DL-101-182-111
+   :language: console
+   :workdir: catalog
+   :cast: catalog_basics
+   :notes: Display the content of the default config file
+
+   $ cat data-cat/config.json | jq .
+
+If no config file is supplied to the ``create`` subcommand, the default is used.
+
+Let's create a new toy catalog with a new config, specifying a new name, a new logo, and new colours for the links.
+This will be the content of the config file, in ``YAML`` format:
+
+.. runrecord:: _examples/DL-101-182-112
+   :language: console
+   :workdir: catalog
+   :cast: catalog_basics
+   :notes: Add a custom config file
+
+   $ cat << EOT >> cat_config.yml
+   # Catalog properties
+   catalog_name: "Toy Catalog"
+   
+   # Styling
+   logo_path: "datalad_logo_funky.svg" # path to logo
+   link_color: "#32A287" # hex color code
+   link_hover_color: "#A9FDAC" # hex color code
+   
+   # Handling multiple metadata sources
+   property_source:
+     dataset: {}
+   
+   EOT
+
+We'll ensure that the new custom logo is available locally:
+
+.. runrecord:: _examples/DL-101-182-113
+   :language: console
+   :workdir: catalog
+   :cast: catalog_basics
+   :notes: Get the custom logo
+
+   $  wget -O datalad_logo_funky.svg https://raw.githubusercontent.com/datalad/tutorials/5e5fc0a4761248e5cedbc491c357d423d14a2b21/notebooks/catalog_tutorials/test_data/datalad_logo_funky.svg
```

Comment:
Consider add `-q`. The timestamps and download performance stats will be different on each run.